lms/envs/test.py

```diff
@@ -582,3 +582,7 @@
 COURSE_CATALOG_API_URL = 'https://catalog.example.com/api/v1'
 
 COMPREHENSIVE_THEME_DIRS = [REPO_ROOT / "themes", REPO_ROOT / "common/test"]
+
+ENV_TOKENS = {
+    'LMS_ROOT_URL': 'https://localhost:8000',
+}
```

Comment:
In that case i should add a corresponding `LMS_ROOT_URL` in `lms/envs/aws.py` and `cms/envs/aws.py` with the value from `ENV_TOKENS['LMS_ROOT_URL']` ?